mailet/base/src/test/java/org/apache/mailet/base/MailetUtilTest.java

```diff
@@ -85,6 +92,87 @@ public void getInitParameterShouldReturnAbsentWhenNull() {
         assertThat(MailetUtil.getInitParameter(mailetConfig, A_PARAMETER)).isAbsent();
     }
 
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerShouldThrowOnEmptyString() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("");
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerShouldThrowOnNull() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger(null);
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerShouldThrowOnInvalid() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("invalid");
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerShouldThrowOnNegativeNumber() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("-1");
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerShouldThrowOnZero() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("0");
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerShouldParseCorrectValue() throws Exception {
+        assertThat(MailetUtil.getInitParameterAsStrictlyPositiveInteger("1"))
+            .isEqualTo(1);
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerWithDefaultValueShouldThrowOnEmptyString() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("", DEFAULT_VALUE);
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerWithDefaultValueShouldThrowOnNull() throws Exception {
+        assertThat(MailetUtil.getInitParameterAsStrictlyPositiveInteger(null, DEFAULT_VALUE))
+            .isEqualTo(DEFAULT_VALUE);
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerWithDefaultValueShouldThrowOnInvalid() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("invalid", DEFAULT_VALUE);
+    }
+
+    @Test
+    public void getInitParameterAsStrictlyPositiveIntegerWithDefaultValueShouldThrowOnNegativeNumber() throws Exception {
+        expectedException.expect(MessagingException.class);
+
+        MailetUtil.getInitParameterAsStrictlyPositiveInteger("-1");
```

Comment:
The name testing function said "WithDefaultValue..." Does it expect one more parameter value for default?